README.md

```diff
@@ -40,6 +40,19 @@ $ cd cardano-node
 $ cabal build
 ```
 
+## Windows Executable
+
+### Download
+You can download [here](https://hydra.iohk.io/job/Cardano/cardano-node/cardano-node-win64/latest-finished).
+
+### Instructions
+
+The download includes cardano-node.exe and a .dll. To run the node with cardano-node run you need to reference a few files and directories as arguments. These can be copied from the cardano-node repo into the executables directory. The command to run the node on mainnet looks like this:
+
+```
+cardano-node.exe run --topology ./mainnet-topology.json --database-path ./state --port 3001 --config ./configuration-mainnet.yaml --socket-path ./socket/mainnet-socket
+```
+
```

Comment:
If we want users to be able to run it on the testnet then maybe some files in `configuration` directory should be renamed ? 
It's clear what files to pickup for mainnet as topology and config files are paired with the same `mainnet` word in their names: `configuration-mainnet.yaml` & `mainnet-topology.json` but for testnet I think there are no such pairs and it can be cryptic for the end user.

Reply:
One more thing is that in order to run node user currently needs to have also `configuration` directory in the same place where binary is placed, and inside it genesis file for `mainnet`:

`C:\Users\IEUser\Downloads\cardano-node-1.9.0-win64>cardano-node.exe run --database-path db --topology mainnet-topology.json --socket-path socket/mainnet-socket --config configuration-mainnet.yml --port 7776`

> cardano-node.exe: configuration/mainnet-genesis.json: openBinaryFile: does not exist (No such file or directory)